run_tests.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+import robotpy.main
```

Comment:
Hm?

Reply:
I was experimenting to see if I could access the robotpy installer through the library